core/scheduler/src/main/scala/org/apache/openwhisk/core/scheduler/FPCSchedulerServer.scala

```diff
@@ -41,10 +43,28 @@ class SchedulerServer(scheduler: SchedulerCore, systemUsername: String, systemPa
   override def routes(implicit transid: TransactionId): Route = {
     super.routes ~ extractCredentials {
       case Some(BasicHttpCredentials(username, password)) if username == systemUsername && password == systemPassword =>
-        (path("disable") & post) {
+        (path("state") & get) {
+          complete {
+            scheduler.getState.map {
+              case (list, creationCount) =>
+                (list
+                  .map(scheduler => scheduler._1.asString -> scheduler._2.toString)
+                  .toMap
+                  ++ Map("creationCount" -> creationCount.toString)).toJson.asJsObject
+            }
+          }
+        } ~ (path("disable") & post) {
           logger.warn(this, "Scheduler is disabled")
           scheduler.disable()
           complete("scheduler disabled")
+        } ~ (path("queue" / "total") & get) {
```

Comment:
let's put `queue` in a constant as it will likely be reused as new api's are added

Reply:
Updated accordingly, thanks for your review!